dump/mapping/bufr_adpupa_prepbufr.py

```diff
@@ -0,0 +1,291 @@
+#!/usr/bin/env python3
+import os
+import sys
+sys.path.append('/scratch1/NCEPDEV/da/Praveen.Singh/HERA/bufr-query/build/lib')
+sys.path.append('/scratch1/NCEPDEV/da/Praveen.Singh/HERA/bufr-query/build/lib/python3.10')
+sys.path.append('/scratch1/NCEPDEV/da/Praveen.Singh/HERA/bufr-query/build/lib/python3.10/site-packages')
+sys.path.append('/scratch1/NCEPDEV/da/Praveen.Singh/HERA/bufr-query/build/lib/python3.10/site-packages/bufr')
+import bufr
+import argparse
+import copy
+import numpy as np
+import numpy.ma as ma
+import math
+import calendar
+import time
+from datetime import datetime
+from pyioda.ioda.Engines.Bufr import Encoder as iodaEncoder
+from bufr.encoders.netcdf import Encoder as netcdfEncoder
+from wxflow import Logger
+
+# Initialize Logger
+# Get log level from the environment variable, default to 'INFO it not set
+log_level = os.getenv('LOG_LEVEL', 'INFO')
+logger = Logger('bufr_adpupa_prepbufr.py', level=log_level, colored_log=False)
+
+
+def logging(comm, level, message):
```

Comment:
Should we put this method into a util file and import it from the util file? I guess we need to use it many times.

Reply:
It's been lightly discussed but I don't think we fully decided to do this, though it may make sense.